code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -169,21 +169,21 @@
 
 /datum/action/innate/cult/blood_spell/shackles
 	name = "Shadow Shackles"
-	desc = "A stealthy spell that will handcuff and temporarily silence your victim."
+	desc = "Empowers your hand to start handcuffing victim on contact, and mute them if successful."
 	button_icon_state = "cuff"
 	charges = 4
 	magic_path = "/obj/item/melee/blood_magic/shackles"
 
 /datum/action/innate/cult/blood_spell/construction
 	name = "Twisted Construction"
-	desc = "<u>A sinister spell used to convert:</u><br>Plasteel into runed metal<br>50 metal into a construct shell<br>Cyborgs directly into constructs<br>Cyborg shells into construct shells<br>Airlocks into runed airlocks (harm intent)"
+	desc = "Empowers your hand to corrupt certain metalic objects. Examine in hand for more information."	
```

Comment:
i feel like the old description would be more useful for someone who didnt know what this did